docs/_i18n/en/documentation/table-options.md

```diff
@@ -70,6 +70,13 @@ The table options are defined in `jQuery.fn.bootstrapTable.defaults`.
         <td>Defines the column sort order, can only be 'asc' or 'desc'.</td>
     </tr>
     <tr>
+        <td>rememberOrder</td>
+        <td>data-remember-order</td>
+        <td>Boolean</td>
+        <td>true</td>
```

Comment:
I feel like everyone is going to want to set this option to false, and this is very subtle behavior that many people won't even realize that they want. I realize it would be a "breaking" change, but how about defaulting this option to false? I personally don't see any value in this option, so I'd even vote for removing it entirely and unconditionally switching to the new behavior.